calcom/cal.com
# Show cancel an reschedule option for bookings that are still waiting for confirmation

## What does this PR do?

When a user that booked a meeting that requires confirmed clicked on the cancel link in the confirmed email. It redirected to this page without any option to cancel the booking: 
![Screenshot 2022-11-29 at 13 48 39](https://user-images.githubusercontent.com/30310907/204533276-7e632351-2564-4ddc-9292-4269fe5d3684.png)

This PR shows the cancel (and reschedule option) also for bookings that are still waiting for confirmation (only shows it for user that is not the organizer or the meeting)

![Screenshot 2022-11-29 at 13 53 15](https://user-images.githubusercontent.com/30310907/204534269-cd49f2cc-5d48-4fba-815e-18943220a076.png)

**Environment**: Staging(main branch)

## Type of change


## How should this be tested?

- Book an event type that requires confirm
- Click on the cancel link in the confirmation email (don't be logged in as the organizer of the meeting)
- See that you can now cancel the booking
